Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:bullseye-slim
+
+WORKDIR /usr/src/app
+
+RUN apt-get update \
+     && apt-get install -y --no-install-recommends build-essential jq liblmdb-dev lmdb-utils curl libcurl4-openssl-dev ca-certificates openssl libmicrohttpd12 libmicrohttpd-dev \
```

Comment:
is `jq` really required? I don't see where ...

Reply:
This is used in the shell scripts (get-stats.sh and c-mini-test.sh). When inside the container it is convenient to have these.